src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReader.netstandard.cs

```diff
@@ -36,6 +42,48 @@ internal AssemblyName GetAssemblyName(StringHandle nameHandle, Version version,
             return assemblyName;
         }
 
+        internal static unsafe AssemblyName GetAssemblyName(string assemblyFile)
```

Comment:
I am surprised that the linker does not trim this and all tests work.

Reply:
I belive that's because there is IVT in this assembly